bootstrap-pull-request/src/prebuilt.ts

```diff
@@ -13,7 +14,45 @@ type Inputs = {
   substituteVariables: Map<string, string>
 }
 
-export const copyServicesFromPrebuilt = async (inputs: Inputs): Promise<void> => {
+export const syncServicesFromPrebuilt = async (inputs: Inputs): Promise<void> => {
+  // If an application has been removed from the prebuilt branch,
+  // it will remain in the existing namespace branch.
+  //
+  // To clean up the outdated applications,
+  // deleteExistingApplications() deletes the existing applications, and then
+  // copyServicesFromPrebuilt() recreates the applications.
+  await deleteExistingApplications(inputs)
+  await copyServicesFromPrebuilt(inputs)
```

Comment:
`copyServicesFromPrebuilt` is the original function.
In this pull request, I have added `deleteExistingApplications`.